configure

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+AUTO_QMAIL=/var/qmail
+
+for arg in $*; do
+	case "${arg}" in
+	--prefix=*)
+		AUTO_QMAIL=$(echo "${arg}" | sed 's#^--prefix=##')
+		;;
+	--help)
+		echo "--prefix    install prefix [/var/qmail]"
```

Comment:
if we document `--prefix` and not `--prefix=dir`, we would have to support `--prefix dir`?

Reply:
good catch